### ISSUE TITLE
Update installation guide

### ISSUE THREAD

**Issue by Kevgilmore** [GluuFederation/k8s-gluu-server#3]
I'm trying to install gluu server by following the installation instructions in the readme.

It says "Create a virtual machine using virtualbox. Get its ip. Then run this cmd inside new vm:"

What OS should this vm run? 
How should it be setup etc?

 At the moment I'm stuck and can't open the gui, all I've found is http://192.168.99.100:32749/ui/#/dc1/services - which takes me to some bare looking ui.

Basically, it's incomplete and not clear how to get this setup and running.

Thanks

**Comment by willow9886** [comment]
Yes, this is a skunks works project and not ready for use. Feel free to continue to tinker, but we don't have the bandwidth to support it right now. Hopefully we will cycle back later this year.
